### ISSUE TITLE
Create new column in the database and an email tag

### ISSUE THREAD

**Issue by joybuchanan** [rememberlenny/national-fever-tracker#7]
In the data we save each day, indicate whether a report came in from an email or through the website form. Binary indicator for each row.

Reason: data coming in from daily emails will be of higher quality

**Comment by joybuchanan** [comment]
We are considering security. The vision is that users will check in exactly once per day and report the most extreme symptoms they experienced in the last 24 hours. Ideally, the text of the website and functionality will reflect this.

**Comment by jrhorn424** [comment]
My suggestion for this feature is using a tracking parameter in the e-mail. Postmark should be able to provide these, or we can do it in the application. Results coming from e-mail will use this parameter in the URL, and we can pass that along to the server using a hidden input in the form.

**Comment by joybuchanan** [comment]
This sounds good to me.

Right now something we are not doing, is we do not have the email address
as a column in the main database. The symptom data is divorced from
individually identifying information. We don't even have to worry too much
about security, since the database has nothing sensitive in it.

I assume that, if we do what you propose, this would still be true?

**Comment by jrhorn424** [comment]
Yes, nothing regarding security changes. In fact, nothing about the way we structure our data internally (in terms of what's divorced from what; we call that "normalization") affects security. Storing e-mail is a concern from a data-leak perspective, but the e-mail addresses are not considered NPPI (https://www.stevens.edu/directory/information-technology/cyber-security-and-information-security/what-nppi)

**Comment by joybuchanan** [comment]
Cool.
After we settle on something, it would be nice to have brief notes about it
in the Slack, too.

**Comment by joybuchanan** [comment]
And feel free to make a new Slack channel if you have details to go into.

**Comment by joybuchanan** [comment]
Also, I know you already saw this, but Lenny said that there is no need to
log into PostMark to change email settings. It's all done through the code,
which should all be on the repo.
